apps/frontend/src/types/assets.ts

```diff
@@ -0,0 +1,6 @@
+export enum Asset {
+  RBTC = 'RBTC',
+  ZUSD = 'ZUSD',
+  XUSD = 'XUSD',
+  DLLR = 'DLLR',
+}
```

Comment:
Use tokens from https://github.com/DistributedCollective/sovryn-dapp/pull/92 instead.
There is enum `SupportedTokens` in `@sovryn/contracts` package and also helper function `getTokenDetails` which will return symbol name (and some additional info).

Ofc you will need to add RBTC, ZUSD and DLLR tokens there to use them.

After that, instead of using this `Asset` enum have some constant with array of `SupportedTokens` inside of dapp, to filter out tokens that should be visible in WalletBalance component